pkg/ebpf/c/tracer-ebpf.c

```diff
@@ -705,6 +705,18 @@ int kprobe__udp_recvmsg(struct pt_regs* ctx) {
     return 0;
 }
 
+SEC("kprobe/udp_recvmsg/rhel")
+int kprobe__udp_recvmsg_rhel(struct pt_regs* ctx) {
+    struct sock* sk = (struct sock*)PT_REGS_PARM2(ctx);
+    u64 pid_tgid = bpf_get_current_pid_tgid();
+
+    // Store pointer to the socket using the pid/tgid
+    bpf_map_update_elem(&udp_recv_sock, &pid_tgid, &sk, BPF_ANY);
+    log_debug("kprobe/udp_recvmsg/rhel: pid_tgid: %d\n", pid_tgid);
```

Comment:
do we need to ship this?

Reply:
`log_debug` is behind a macro and we generate two versions of the ELF file: one w/ logging and one without.

Reply:
Sure.  My unstated concern was that it would be too busy of a log line when debugging is enabled but it looks like we have similar debug logs for other kprobes